File: Napakalaki Java/src/NapakalakiGame/Cultist.java
/**
 *
 * @author Jesús Sánchez de Lechina Tejada & Miguel Ángel Robles Urquiza
 */
package NapakalakiGame;

public class Cultist {
    private String name;
    private int gainedLevels;
    
    public Cultist(String name, int gainedLevels) {
        this.name = name;
        this.gainedLevels = gainedLevels;
    }
    
    public int getGainedLevels(){
        return gainedLevels;
    }
}

File: Napakalaki Ruby/lib/cultist.rb
# By: Miguel Robles Urquiza
#     Jesús Sánchez de Lechina Tejada

module NapakalakiGame

  class Cultist

    def initialize(name, gainedLevels)
      @name = name
      @gainedLevels = gainedLevels

    end
    
    def to_s
      return "#{@name} (CombatLevel +#{@gainedLevels}) "
    end

    def getGainedLevels
      @gainedLevels
    end
  end
  
end
